### PR TITLE
Iss3: Improves the jupyter notebooks for data analysis

### LINKED ISSUE #3: Move Colab Notebooks to Jupyter Notebooks
This issue can be closed when the analysis done in Google Colab has been moved Jupyter Notebooks.

### PR DESCRIPTION
**Pull request by samgdotson**
Closes #3 by converting all google colab notebooks to jupyter notebooks that can be run immediately after following instructions in the README.

### CODE REVIEW

File: data_processing/grid_demand.ipynb [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = '../../data/'\n",
```
Comment (robfairh) [inline data_processing/grid_demand.ipynb:30]: Same comment as before, can you clarify how to access the data? I would add a markdown cell that clarifies that.

File: data_processing/solar_power.ipynb [hunk 2]
```diff
@@ -0,0 +1,1937 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define path to data\n",
+    "path = \"../../data/\" "
```
Comment (robfairh) [inline data_processing/solar_power.ipynb:54]: Same comment

File: data_processing/steam_demand.ipynb [hunk 3]
```diff
@@ -0,0 +1,838 @@
...
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "steam15[(steam15['eDNA Status as String'] == 'UNRELIABLE') & (steam15.index.month <= 1)].plot()"
```
Comment (robfairh) [inline data_processing/steam_demand.ipynb:91]: What's this for? Does it plot just the unreliable data?

**Review by robfairh — CHANGES_REQUESTED** [review]
See the comments about clarifying how to access the data.

File: data_processing/steam_demand.ipynb (hunk 3 above)
Comment (samgdotson) [reply data_processing/steam_demand.ipynb:91]: Yes, it plots all of the data, and then it plots the data points marked as "unreliable." I wanted to see if the "unreliable" data was really that far off the mark. It seems that most of them are false positives.

**Comment by samgdotson** [comment]
The graph titled "checking unreliable data" is only for visual confirmation that the data is reasonable. 
The blue line is the measured value, the red dots are the values marked unreliable. It appears that most of the data is reliable and the "unreliable" tag from the eDNA system (where the data originated) were false positives.

**Pull request merged**